spec/models/manageiq/providers/hawkular/inventory/parser/availability_updates_spec.rb

```diff
@@ -50,6 +50,33 @@
       expect(persister.middleware_servers.size).to be_zero
     end
 
+    it "must create an item in persister with new status data for each domain reported by collector" do
```

Comment:
As per http://www.betterspecs.org/ , I think it is best that this test name reads like:

`it "creates an item in persister with new status data for each domain reported by collector"`

The same for the other tests.

Reply:
Let me change this in another PR since I was quite following the specs already implemented in same file. Better to split it up and clean it afterwards. Anyway, fan of betterspecs.org <3

Reply:
Same for the next changes regarding style and standards.